forge-gui/res/cardsfolder/upcoming/tolsimir_midnights_light.txt

```diff
@@ -0,0 +1,12 @@
+Name:Tolsimir, Midnight's Light
+ManaCost:2 G W W
+Types:Legendary Creature Elf Scout
+PT:3/2
+K:Lifelink
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigToken | TriggerDescription$ When CARDNAME enters the battlefield, create Voja Fenstalker, a legendary 5/5 green and white Wolf creature token with trample.
+SVar:TrigToken:DB$ Token | TokenAmount$ 1 | TokenScript$ voja_fenstalker | TokenOwner$ You
+T:Mode$ Attacks | ValidCard$ Wolf.YouCtrl | IsPresent$ Card.Self+attackedThisCombat | TriggerZones$ Battlefield | Execute$ TrigForceBlock | TriggerDescription$ Whenever a Wolf you control attacks, if NICKNAME attacked this combat, target creature an opponent controls blocks that Wolf this combat if able.
```

Comment:
I know it's intervening if but it it can use LKI for that part anyway, so you should use NoResolvingCheck